extensions/amp-story/1.0/page-advancement.js

```diff
@@ -474,6 +477,15 @@ class ManualAdvancement extends AdvancementConfig {
       return false;
     }
 
+    if (
+      target.getAttribute('has-tooltip') === 'auto' &&
+      this.isInScreenBottom_(target, pageRect)
```

Comment:
The one variation we discussed on the the issue is whether to measure the target or if instead we should just look at the event's `clientX/Y`.

Otherwise, another nuance here is that `target` is not necessarily an `a` tag, but could be a child inside `<a>`, e.g. a `span` inside `<a><span></span></a>`.

Reply:
Using the target rather than `clientY` is a good idea, as we want clicks on a given element to always have the same behavior. Having an element that can either show a tooltip or navigate depending on where you tap on it would be very confusing.
However, I do agree that we need to find a fix for the targetting issue you mentioned.

Reply:
Since this click event listener is initialized with the `capture` option, we will always get the `<a>` tag, even if the element has a child that was clicked.

https://github.com/ampproject/amphtml/blob/e22492b5bad5dadcd229d5a29dbcaafbe650cadf/extensions/amp-story/1.0/page-advancement.js#L311-L314

Reply:
I believe that `this.element_` here refers to the `<amp-story>` tag. 
So `event.currentTarget` would be `amp-story`, and `event.target` could be either `<a>` or anything nested into `<a>` like Dima mentioned.

Reply:
My bad, I had a wrong idea. I'll update this in a bit to handle the nesting issue Dima raised.

Reply:
If I understand correctly, you're suggesting that we should no longer use this 20% bottom of the screen area protection, but some other heuristic? What do you have in mind?

My personal opinion is that this direct 1 tap links only exist for legacy reasons. If we had remote page attachments from the beginning, we would not have implemented them.
This PR introduces a change that allows creation tools and publishers to implement 1 tap links in an easier way, but I personally am not for introducing new UX hacks to work around the first hack (20% safe area) we introduced.

Reply:
No. I'm proposing to keep 20% as is (or at least separate this question entirely). What i'm saying is this: the "auto-1tap link" in the grid layer is the new concept. It's currently disallowed and thus it's a new feature with `<a has-tooltip=auto>` markup. As such we might be able to naturally define exactly what we mean by "auto-1tap". We could define this mode as:

"The format will make the judgement whether the user's click is ambiguous. If it is, the format will prompt user with the tooltip. Putting the links lower to the bottom edge reduces the ambiguity and thus reduces the probability that the link might get prompted".

And if that model works, then implementation could simply be: `showTooltip = mousEvent.clientX < 80vh`

Reply:
Dima, can we move this conversation to a different ticket to unblock this PR? The current implementation gets us to feature parity, as `cta-layer` would not let you overflow from the 20% safe area. If we need further changes, we should review them with more team members.

Resolving this conversation as per Enrique's message that solves the targetting issue with the use of `pointer-events`.

Reply:
I thought about this a little more and I think you're right. I'll take off this issue from this PR. Mainly because the current solution is a 100% superset over clientX/Y, so we can always relax this later. Please continue the review and disregard this thread.